### PR TITLE
Remove decorators and minor fix in ask.py

### PR DESCRIPTION
**Pull request by bhaveshAn**
decorators has been removed and change cls to self made in `ask.py`
@mariobehling Kindly review !
`codacy` showing fail due the pep8/ flake8 issue which is cover in other PR #200

### CODE REVIEW

**Comment by cclauss** [comment]
Are these changes still needed?

**Comment by AnshulMalik** [comment]
@cclauss I don't think these are required any more.

File: app/scrapers/ask.py [hunk 1]
```diff
@@ -4,12 +4,10 @@
 
 class Ask:
     """Scrapper class for Ask"""
-    @classmethod
-    def __init__(cls):
+    def __init__(self):
```
Comment (cclauss) [inline app/scrapers/ask.py]: When you do not use __self__ in a method then it is right to decorate it with [@classmethod](https://docs.python.org/3/library/functions.html#classmethod) and various linters will flag it if you do not.

File: app/scrapers/ask.py [hunk 2]
```diff
@@ -4,12 +4,10 @@
...
         pass
 
-    @classmethod
-    def get_page(cls,query):
+    def get_page(self,query):
```
Comment (cclauss) [inline app/scrapers/ask.py]: When you do not use __self__ in a method then it is right to decorate it with [@classmethod](https://docs.python.org/3/library/functions.html#classmethod) and various linters will flag it if you do not.

File: app/scrapers/ask.py [hunk 3]
```diff
@@ -20,14 +18,13 @@ def get_page(cls,query):
         response = requests.get('http://ask.com/web', headers=header, params=payload)
         return response
 
-    @classmethod
-    def results_search(cls,query):
+    def results_search(self,query):
```
Comment (cclauss) [inline app/scrapers/ask.py]: When you do not use __self__ in a method then it is right to decorate it with [@classmethod](https://docs.python.org/3/library/functions.html#classmethod) and various linters will flag it if you do not.

File: app/scrapers/ask.py [hunk 4]
```diff
@@ -20,14 +18,13 @@ def get_page(cls,query):
...
         """ Search ask for the query and return set of urls
         Returns: urls (list)
                 [[Tile1,url1], [Title2, url2],..]
         """
         urls=[]
-        response = get_ask_page(query)
+        response = self.get_page(query)
```
Comment (cclauss) [inline app/scrapers/ask.py]: This was fixed and merged in #197

**Comment by AnshulMalik** [comment]
@mariobehling This can be closed considering #197

**Comment by cclauss** [comment]
@bhaveshAn Can you please close this PR?

**Pull request closed without merging**